src/components/plots/xyPlot.ts

```diff
@@ -48,6 +39,12 @@ module Plottable {
       if (this._autoAdjustYScaleDomain) {
         this._updateYExtentsAndAutodomain();
       }
+
+      // TODO: is this needed?
+      if (xScale != null) {
+        xScale.onUpdate(this._adjustYDomainOnChangeFromXCallback);
+      }
```

Comment:
Adding the callback needs to happen in the property-setter. If it's not happening in any of the other lines in this method, we should do it here.